pom.xml

```diff
@@ -2586,6 +2591,100 @@
         <zookeeper.version>3.4.6</zookeeper.version>
         <curator.version>2.6.0</curator.version>
       </properties>
+      <dependencyManagement>
+        <dependencies>
+          <!--
+          the AWS module pulls in jackson; its transitive dependencies can create
+          intra-jackson-module version problems.
+          -->
+          <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aws</artifactId>
```

Comment:
This shouldn't need to be in the main pom right? this looks like it's repeating what the cloud module does. It's not reused elsewhere.